app/views/games/show.html.erb

```diff
@@ -1,97 +1,105 @@
 <div class ="game-show">
-  <div>
-    <div id="home-slideshow" class="carousel slide" data-ride="carousel">
-      <% if @game.pictures.length.zero? %>
-        <div class="carousel-inner" role="listbox">
-          <%= image_tag "fallback/default.png", class: "carousel-image" %>
-        </div>
-      <% else %>
-        <ol class="carousel-indicators">
-          <% @game.pictures.each_with_index do |pic, i| %>
-            <li data-target="#home-slideshow" data-slide-to= <%= "#{i}" %>
-              <% if i == 0 %>
-                class="active">
-              <% else %>
-                class="">
-              <% end %>
-            </li>
+  <div class="container-fluid">
+  <div class="row">
+    <div class="col-md-6 col-sm-6">
+      <h2 style="font-weight:bold; margin-bottom: 20px; color: #EF7030;"><%= @game.title %></h2>
```

Comment:
Avoid inlining styles. Can put this in a class? You could use a selector such as `.game-show h2` if it makes sense for the page seeing how this `h2` is inside `div.game-show`.